x/paramHub/types/types.go

```diff
@@ -71,6 +71,12 @@ var (
 
 		"crossDistributeRewardRelayFee":      {},
 		"crossDistributeUndelegatedRelayFee": {},
+
+		"create_validator_open": {},
```

Comment:
Can we change this create_validator_open edit_validator to keep the same style? Same here

Reply:
it's intended to align with `side_` series.

```
		"side_create_validator": {},
		"side_edit_validator":   {},
		"side_delegate":         {},
		"side_redelegate":       {},
		"side_undelegate":       {},

```